pkg/apb/deprovision.go

```diff
@@ -6,7 +6,9 @@ import (
 )
 
 // Deprovision - runs the abp with the deprovision action.
-func Deprovision(instance *ServiceInstance, clusterConfig ClusterConfig, log *logging.Logger) (string, error) {
+func Deprovision(
+	instance *ServiceInstance, clusterConfig ClusterConfig, log *logging.Logger,
+) (string, error) {
```

Comment:
Agreed, I know we've had the same conversation about other PRs. Definitely makes it easier to review.